src/ostorlab/runtimes/local/agent_runtime.py

```diff
@@ -0,0 +1,153 @@
+"""Runtime agent module gives access to methods to create the specific agent configurations & run the agent.
+
+Usage
+    agent_runtime = AgentRuntime(agentSettings, runtime_name, docker_client, mq_service)
+    agent_service = agent_runtime.create_agent_service(network_name, extra_configs)
+"""
+from typing import List, Optional
+import docker
+from docker.types import services as docker_types_services
+import io
+
+from ostorlab.runtimes.local import runtime
+from ostorlab.runtimes import definitions
+from ostorlab.agent import definitions as agent_definitions
+from ostorlab.runtimes.local.services import mq
+
+
+
+class AgentRuntime:
+    """Methods to consolidate the agent settings and agent default defintion, and create the agent service."""
+
+    def __init__(self,
+                 agent: definitions.AgentSettings,
```

Comment:
```suggestion
                 agent_settings: definitions.AgentSettings,
```

Reply:
what about the attribute itself ? i leave it as agent, or update it too ?